### PR TITLE
Finish First part of CMS

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Update homepage festival
Update CMS

### CODE REVIEW

File: apps/cms/src/app/forms/text/text.component.html [hunk 1]
```diff
@@ -2,18 +2,17 @@
...
+    <textarea matInput [formControl]="form" [placeholder]="schema.placeholder"></textarea>
   </ng-container>
 
   <ng-template #short>
-    <input matInput
-      [type]="schema.type"
-      [formControl]="form"
-      [placeholder]="schema.placeholder"
-    />
+    <!-- We need to set the type like that else type is not reflected on FormControl -->
```
Comment (fritzschoff) [inline apps/cms/src/app/forms/text/text.component.html:9]: wouldn't be a switch case cleaner?

File: apps/cms/src/app/home/slider/slider.component.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  select(event: MatRadioChange) {
+    this.mode = event.value;
+    this.selectForm();
+  }
+}
+
+
+@NgModule({
```
Comment (fritzschoff) [inline apps/cms/src/app/home/slider/slider.component.ts:66]: export this into its own file?

File: apps/festival/festival/src/app/marketplace/home/home.module.ts [hunk 3]
```diff
@@ -17,19 +17,22 @@ import { MovieCardModule } from '@blockframes/movie/components/card/card.module'
...
+import { ImageReferenceModule } from '@blockframes/media/directives/image-reference/image-reference.module';
 
 // Page
 import { HomeComponent } from './home.component';
+import { HomeQueryTitlesPipe, HomeQueryOrgsPipe, HomeGetOrgPipe } from './home.pipe';
 
 
 @NgModule({
-  declarations: [HomeComponent],
+  declarations: [HomeComponent, HomeQueryTitlesPipe, HomeQueryOrgsPipe, HomeGetOrgPipe],
```
Comment (fritzschoff) [inline apps/festival/festival/src/app/marketplace/home/home.module.ts:28]: this will break when you import the pipes somewhere else

**Review by fritzschoff — APPROVED** [review]

File: apps/cms/src/app/home/slider/slider.component.ts (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/cms/src/app/home/slider/slider.component.ts:66]: No I need to have it in the component file for lazy loaded component

File: apps/festival/festival/src/app/marketplace/home/home.module.ts (hunk 3 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/marketplace/home/home.module.ts:28]: I don't plan to use the pipes somewhere else. They are specific to this component.

**Pull request merged**